charts/private/cert-manager/values.yaml

```
@@ -1,3 +1,7 @@
 cert-manager:
   namespace: cert-manager
   installCRDs: true
+letsEncrypt:
+  email: <YOUR_LETS_ENCRYPT_EMAIL> # Required: Let's Encrypt recipient email.
+cloudflare:
+  apiToken: <YOUR_CLOUDFLARE_API_TOKEN> # Required: Cloudflare account API token.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)